package com.qa.exceptionsexercise;
public class Main {
public static void main(String[] args) {
	System.out.println(ExceptionGenerator.divideNumbers(2, 4));
}
}
